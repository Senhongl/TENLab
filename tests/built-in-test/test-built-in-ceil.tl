a = [[10,11,12],[14,15,16]];
print(ceil(a));
b = [1.0,3.2,5.6];
print(ceil(b));