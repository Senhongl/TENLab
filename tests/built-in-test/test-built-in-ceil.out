 10  11  12
 14  15  16
[ CPUDoubleType{2,3} ]
 1
 4
 6
[ CPUDoubleType{3} ]
